README.md

```diff
@@ -142,7 +142,17 @@ export SALESFORCE_API_VERSION="41.0"
 client = Restforce.new
 ```
 
-### Proxy Support
+#### Sandbox Orgs
+
+You can connect to sandbox orgs by specifying a host. The default host is
+'login.salesforce.com':
+
+```ruby
+client = Restforce.new(host: 'test.salesforce.com')
+```
+The host can also be set with the environment variable `SALESFORCE_HOST`.
+
+#### Proxy Support
```

Comment:
I changed this from an h3 to h4 since it seemed on the same level as the other option documentatation.